jussi/errors.py

```diff
@@ -237,3 +237,18 @@ class UpstreamResponseError(JsonRpcError):
 class InvalidNamespaceError(JsonRpcError):
     code = 1200
     message = 'Invalid JSONRPC method namespace {namespace}'
+
+
+class InvalidNamespaceAPIError(JsonRpcError):
+    code = 1200
+    message = 'Invalid JSONRPC method namespace, unable to resolve {namespace}.{api}'
```

Comment:
Nice!

Reply:
If I grokked the steem code correctly, 0 will always be `database_api` and 1 will always be `login_api`, and the other numerical api specifiers are not fixed, and, therefore, cannot be supported.  Did I get that correct?

Reply:
Yeah sounds right, I think the numbers correspond to the order the plugins are loaded.